DOCS/man/input.rst

```diff
@@ -3438,6 +3444,35 @@ Property list
 ``current-ao``
     Current audio output driver (name as used with ``--ao``).
 
+``clipboard`` (RW)
+    Access to the system clipboard as a key/value map of types and data.
+
+    Sub-paths can be accessed directly; e.g. ``clipboard/text`` can be read, written,
+    or observed.
+
+    The top-level object itself can be written directly as a shortcut; the intended
+    type will be guessed.
+
+    Converting this property to a string will give a JSON representation of all types.
+    Unset types will be null.
+
+    Writing to one type may cause other types to also be updated.
+
+    Currently, the following types are available:
+
+    ``text``
+        Arbitrary plain text
+
+    ``url``
+        A URL; if ``path`` is populated, this will be a corresponding file:// URL
+
+    ``paths``
+        A list of absolute paths to files on disk.
+
+    ``path``
+        A single absolute path to a file on disk.
+        Exposed for convenience and not included in the top-level map.
```

Comment:
So `get_property_native("clipboard")` will include text, url and path**s** yet `get_property_string("clipboard/path")` will work?
It doesn't seem wise to introduce such behaviour into properties.

Reply:
I could include it; it's just redundant with `paths`, which would make it a little more annoying to use for development/debugging.